torchdiffeq/_impl/adaptive_heun.py

```diff
@@ -0,0 +1,111 @@
+# Based on https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/integrate
+import torch
+from .misc import (
+    _scaled_dot_product, _convert_to_tensor, _is_finite, _select_initial_step, _handle_unused_kwargs, _is_iterable,
+    _optimal_step_size, _compute_error_ratio
+)
+from .solvers import AdaptiveStepsizeODESolver
+from .interp import _interp_fit, _interp_evaluate
+from .rk_common import _RungeKuttaState, _ButcherTableau, _runge_kutta_step
+
+_ADAPTIVE_HEUN_TABLEAU = _ButcherTableau(
+    alpha=[1.],
+    beta=[
+        [1.],
+    ],
+    c_sol=[1 / 2, 1 / 2],
+    c_error=[
+        1. - 1 / 2,
+        0. - 1 / 2,
```

Comment:
Can you simplify these expressions into single floats, e.g. 0.5?

Reply:
Just changed it to single floats.